function-images/tests/video-analytics/knative_yamls/xdt/service-recog.yaml

```diff
@@ -27,6 +27,10 @@ metadata:
   namespace: default
 spec:
   template:
+    metadata:
+      annotations:
+        # set this equal to DecoderFrames in decoder manifest
+        autoscaling.knative.dev/minScale: "6"
```

Comment:
please also set the hard concurrency limit to 1 
https://knative.dev/docs/serving/autoscaling/concurrency/